WBooks/src/app/screens/BookDetail/components/DetailBox/index.js

```diff
@@ -23,45 +21,18 @@ class DetailBox extends Component {
     const rentButtonStyle = isAvailable(styles.rentButtonAvailable, styles.rentButtonNotAvailable);
     const disabled = !genericBook.isAvailable;
     return (
-      <View style={styles.bookContainer}>
-        <View style={styles.detailContainer}>
-          <View>
-            <ImageBook style={styles.image} source={image} />
-          </View>
-          <View style={styles.infoContainer}>
-            <View>
-              <Text style={styles.title}>{title}</Text>
-            </View>
-            <View>
-              <Text style={[availabilityStyle, styles.buttonAndAvailableDesign]}>{availabilityText}</Text>
-            </View>
-            <View>
-              <Text style={styles.textInfo}>{author}</Text>
-            </View>
-            <View>
-              <Text style={styles.textInfo}>{year}</Text>
-            </View>
-            <View>
-              <Text style={styles.textInfo}>{genericBook.type}</Text>
-            </View>
-          </View>
-        </View>
-        <View style={styles.buttonsContainer}>
-          <TouchableOpacity
-            onPress={this.handleAddToWishlist}
-            style={[styles.genericStyleButton, styles.wishlistButton]}
-          >
-            <Text style={[styles.buttonAndAvailableDesign, styles.wishlistText]}>{buttonText.wishlist}</Text>
-          </TouchableOpacity>
-          <TouchableOpacity
-            onPress={this.handleRentBook}
-            style={[styles.genericStyleButton, rentButtonStyle]}
-            disabled={disabled}
-          >
-            <Text style={[styles.genericTextButton, styles.textAvailable]}>{buttonText.rent}</Text>
-          </TouchableOpacity>
-        </View>
-      </View>
+      <DetailBoxLayout
```

Comment:
I will move to the layout file all `availabilityStyle`, `availabilityText `,  `rentButtonStyle ` and `disabled `

Reply:
But that props are logic. I thought that the layout show something and it don't  have logic. Or when the logic is to decide what to show it get into the layout file?